langs/apl/apl

```diff
@@ -0,0 +1,9 @@
+#!/bin/sh -e
+
+export DYALOG=/opt/mdyalog/18.2/64/unicode
```

Comment:
Let me go first. I've a problem with that line. As the version changes, so does the directory. I've attempted patch files, but I already know you'd rather not see them being used. No big deal, because they didn't work anyway. The executable's looking in that particular directory. I've been transferring to `/usr/local/...`, overriding `DYALOG`, and defining an additional entry in `PATH`, but it will just not start looking in places other than `/opt/mdyalog/18.2/64/unicode`.

So yeah, this one annoys me quite a bit in terms of bumping releases.